ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/name/vch-creation-wizard-name.scss

```diff
@@ -1,14 +1,4 @@
-div.form-group {
-  padding-left: 150px;
-  & label:first-child {
-    width: 150px;
-  }
-  & span.red-text {
-    color: red;
-    margin: 0;
-  }
-}
-
-.smaller-p-body-text {
-  font-size: 12px;
+:host /deep/ clr-signpost .signpost-action {
```

Comment:
/deep/ is interpreted by angular cli in build time and rendered with standard css selectors. It allows us to by-pass child components styles encapsulation. Clarity is adding margins to all buttons that are form child elements. Even buttons rendered by Clarity's Signpost components. That's why we need this. I will replace this with its alias (::ng-deep) to make it more specific.